packages/server/graphql/public/types/RRule.ts

```diff
@@ -5,25 +5,31 @@ import {RRuleScalarConfig} from '../resolverTypes'
 const isRRuleValid = (rrule: RRule) => {
   if (rrule.options.freq !== Frequency.WEEKLY) {
     return {
-      error: 'Query error: Weekly is the only supported frequency'
+      error: 'RRule frequency must be WEEKLY'
+    }
+  }
+
+  if (isNaN(rrule.options.interval)) {
```

Comment:
added validation so that won't happen again, ie user will see an error when input is wrong